### PR TITLE
Web worker

### PR DESCRIPTION
**Pull request by imbcmdth**
Preliminary build system for creating a single stand-alone MSE polyfill with an inlined webworker.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -16,9 +81,87 @@ module.exports = function(grunt) {
...
+      }
+
+      // Read file source.
+      var srcContents = grunt.file.read(src);
+      var currentPath = path.dirname(src);
+
+      // Process the file looking for `importScripts` calls
+      grunt.file.write(f.dest, falafel(srcContents, function (node) {
+        // Check every function call for one that calls importScripts
+        if (node.type === 'CallExpression' && node.callee.name === 'importScripts') {
```
Comment (dmlap) [inline Gruntfile.js]: This is pretty cool.

File: test/media-sources_test.js [hunk 2]
```diff
@@ -302,7 +326,11 @@
     this.getFlvHeader = function() {
       return new Uint8Array([1, 2, 3]);
     };
-    this.parseSegmentBinaryData = function(data) {};
+    this.parseSegmentBinaryData = function(data) {
+      tags.push({
```
Comment (forbesjo) [inline test/media-sources_test.js]: Where does `tags` come from?

File: test/media-sources_test.js [hunk 3]
```diff
@@ -228,11 +253,10 @@
   });
 
   module('Flash MediaSource', {
-    setup: function(assert) {
-      var swfObj;
-
```
Comment (forbesjo) [inline test/media-sources_test.js]: `swfObj` and `assert` are still used below.

**Comment by imbcmdth** [comment]
Replaced by #32.

**Pull request closed without merging**